src/sources/mainwindow.cpp

```diff
@@ -13,3 +16,46 @@ MainWindow::~MainWindow()
 {
     delete ui;
 }
+
+void MainWindow::on_actionLoad_Reference_Image_triggered()
+{
+    QString fileName = QFileDialog::getOpenFileName(this,
+                                                     tr("Select Reference Image File"),
+                                                     "",
+                                                     "Images (*.png *.jpg *.tiff);;All Files (*)"
+                                                     );
+    QString status = tr("");
+    if (fileName.length() == 0) status = tr("No File Selected");
+    else status = tr("Selected ") + fileName;
+    statusBar()->showMessage(status);
+}
+
+void MainWindow::on_actionLoad_Current_Image_s_triggered()
+{
+    QStringList files = QFileDialog::getOpenFileNames(this,
+                                                      tr("Select Current Image(s)"),
+                                                      "",
+                                                      "Images (*.png *.jpg *.tiff);;All Files (*)"
+                                                      );
+    QString status = "";
+    switch (files.length()) {
```

Comment:
for multiple images just mention the number of files? Or at the most the first and the last file names.